<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zdeny_PhD_ShenanigansClass</class>
 <widget class="QMainWindow" name="Zdeny_PhD_ShenanigansClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>271</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Zdeny_PhD_Shenanigans</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>211</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>85</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Zdeny's PhD Shenanigans</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>29</y>
      <width>185</width>
      <height>185</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonDebug">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DEBUG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>CLOSE ALL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>215</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuProjects">
    <property name="title">
     <string>Projects</string>
    </property>
    <addaction name="actionIPC"/>
    <addaction name="actionDiffrot"/>
    <addaction name="actionFiltering"/>
    <addaction name="actionFeatures"/>
    <addaction name="actionFITS"/>
    <addaction name="actionProcedural"/>
    <addaction name="actionSnake"/>
    <addaction name="actionUnitTests"/>
   </widget>
   <addaction name="menuProjects"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Zdeny's PhD Shenanigans</string>
   </property>
  </action>
  <action name="actiongamma_contrast_brightness">
   <property name="text">
    <string>gamma / contrast / brightness</string>
   </property>
  </action>
  <action name="actionfourier_filtering">
   <property name="text">
    <string>fourier filtering</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionIPC">
   <property name="text">
    <string>IPC</string>
   </property>
  </action>
  <action name="actionMandelbrot">
   <property name="text">
    <string>mandelbrot</string>
   </property>
  </action>
  <action name="actionANN">
   <property name="text">
    <string>ANN</string>
   </property>
  </action>
  <action name="actionCNN">
   <property name="text">
    <string>CNN</string>
   </property>
  </action>
  <action name="actiontest_benchmark">
   <property name="text">
    <string>test benchmark</string>
   </property>
  </action>
  <action name="actionIPC_optimize">
   <property name="text">
    <string>IPC optimize</string>
   </property>
  </action>
  <action name="actionIPC_duel">
   <property name="text">
    <string>2 IPC duel</string>
   </property>
  </action>
  <action name="actiondiffrot">
   <property name="text">
    <string>Diffrot</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="text">
    <string>Debug</string>
   </property>
  </action>
  <action name="actionFITS_download_url_creator">
   <property name="text">
    <string>FITS download url creator</string>
   </property>
  </action>
  <action name="actionFITS_download_url_creator_2">
   <property name="text">
    <string>FITS download url creator</string>
   </property>
  </action>
  <action name="actionIPC_2pic_align">
   <property name="text">
    <string>IPC 2 pic</string>
   </property>
  </action>
  <action name="actionFeature_matching">
   <property name="text">
    <string>feature matching</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionGenerate_land">
   <property name="text">
    <string>Generate land</string>
   </property>
  </action>
  <action name="actionFits_downloader">
   <property name="text">
    <string>Fits downloader</string>
   </property>
  </action>
  <action name="actionFits_checker">
   <property name="text">
    <string>Fits checker</string>
   </property>
  </action>
  <action name="actionFeature_extract">
   <property name="text">
    <string>Feature extract</string>
   </property>
  </action>
  <action name="actionFeature_match">
   <property name="text">
    <string>Feature match</string>
   </property>
  </action>
  <action name="actionDiffrot">
   <property name="text">
    <string>Diffrot</string>
   </property>
  </action>
  <action name="actionFITS_downloader">
   <property name="text">
    <string>FITS downloader</string>
   </property>
  </action>
  <action name="actionFITS_checker">
   <property name="text">
    <string>FITS checker</string>
   </property>
  </action>
  <action name="actionFeatures">
   <property name="text">
    <string>Features</string>
   </property>
  </action>
  <action name="actionSnake">
   <property name="text">
    <string>Snake</string>
   </property>
  </action>
  <action name="actionProcedural">
   <property name="text">
    <string>Procedural gen</string>
   </property>
  </action>
  <action name="actionFITS">
   <property name="text">
    <string>FITS</string>
   </property>
  </action>
  <action name="actionUnitTests">
   <property name="text">
    <string>Unit tests</string>
   </property>
  </action>
  <action name="actionFiltering">
   <property name="text">
    <string>Filtering</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
